### DISCUSSION TITLE
Release Tunny v0.8.1

### DISCUSSION THREAD

**Discussion by hrntsm** [Tunny-gh/Tunny_in_Open_Sea#186] [Announcements]
## Added

- Debug log output button

## Changed

- Improved stability of python library installation
  - The library is now included from the beginning in whl format and the file is installed, eliminating the need to download it from the Internet.

## What's Changed

* Featura/use local wheel install by @hrntsm in https://github.com/hrntsm/Tunny/pull/184
* Release v0.8.1 by @hrntsm in https://github.com/hrntsm/Tunny/pull/185

**Full Changelog**: https://github.com/hrntsm/Tunny/compare/v0.8.0...v0.8.1

<hr /><em>This discussion was created from the release <a href='https://github.com/hrntsm/Tunny/releases/tag/v0.8.1'>Tunny v0.8.1</a>.</em>

**Comment by mfseingim** [comment]
Hi, I need some help with the installation of Tunny. I downloaded Tunny and I followed all the steps reported in the guide but when I restart Rhino and grasshopper I don't find it. I don't understand why this happens...
Thank you a lot.

**Reply by hrntsm** [reply]
Hi @mfseingim 
Could you try installing v0.8.2 using Rhino's PackageManager?

**Comment by mfseingim** [comment]
Is it possible? I tried to search for Tunny but I didn't find anything...
________________________________

**Reply by hrntsm** [reply]
If you have Rhino 7.13 or higher on windows, it will be displayed. Is your version OK?
